students/sendecki_andrzej/lesson_01_basics/first_digit_after_decimal_point.py

```diff
@@ -0,0 +1,13 @@
+# lesson_01_basics
+# First digit after decimal point
+#
+# Statement
+# Given a positive real number, print its first digit to the right of the decimal point.
+
+print("Enter a positive real number")
+n = float(input())
+
+res = int(n * 10) % 10
+
+print("First digit to the right of '.' is: " + str(res) + ".")
```

Comment:
using `+` for string formatting is OK. However, the preferred way is to use `format`.
Check it out at https://pyformat.info/